charters/Architecture/charter.md

```diff
@@ -0,0 +1,52 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including
+identifying and addressing *cross-cutting* architecture, integration, and operational concerns across DCP
+projects.
+
+The core team is composed of *Technical Leads* representing each technical DCP project group to ensure a diversity of
+skills and views needed for community technical decision-making. 
+
+## In-scope
+
+Software architecture responsibilities
+* Gather and synthesize DCP's end-to-end requirements
+* Set overall direction of architectural and software design direction for the DCP
+* Provide technical input for the PM product roadmap process
+* Review and provide final approval for technical charters for DCP projects
+* Publish Architecture Requests for Comments (RFCs) documenting technical debt that impacts delivery of DCP as an
+  integrated system
+* Review and provide final approval for cross-project RFC(s)
+    * Available to mentor on all technical RFCs
+* Develop architecture standards and best practices including, but not limited to:
```

Comment:
I think between this group and the PM group it's important to keep an eye on what other large consortium are doing.  NIH, EBI, etc all have major cloud infrastructure projects and, as a scientist, being able to work across these environments and access data from HCA, GDC, GTEx, HIVE HubMap, etc would be helpful.  We see this happening with commercial and academic platforms starting to lower barriers for their users to pull data from a different large projects and we'd want the same thing to happen for HCA, e.g. researchers can use HCA data on the cloud in their favorite platform and combine with other interesting datasets.  There are technical coordination that need to happen to make this easier/possible.  Because of this, I think the tech group should have an item in the charter for "monitor and participate in technical coordination with other large projects as needed to facilitate data use".  Or something similar...

Reply:
Resolved with new bullet